themes/nord-color-theme.json

```
@@ -1215,6 +1215,45 @@
         "foreground": "#81A1C1"
       }
     },
+    {
+      "name": "[Rust] Entity types",
+      "scope": [
+        "source.rust entity.name.type"
+      ],
```

```suggestion
      "scope": "source.rust entity.name.type",
```

Lets just use a simple string when there is only one selector.